var/spack/repos/builtin/packages/jali/package.py

```diff
@@ -31,7 +32,8 @@ class Jali(CMakePackage):
 
     depends_on('mpi')
 
-    depends_on('boost')
+    if version < Version('1.1.6'):
```

Comment:
@alalazo pointed out that this needs to be `if self.spec.version < Version('1.1.6')`, or - more idiomatic to spack - `if self.spec.satisfies('@:1.1.5')` (I prefer the latter)

Reply:
Wait scratch all that this occurs outside of the method: so it needs to be

```
depends_on('boost', when='@:1.1.5')
```

(and remove the `if`)

Reply:
ok. Thanks